deps.edn

```diff
@@ -73,13 +73,13 @@
   inflections/inflections                   {:mvn/version "0.14.1"}             ; Clojure/Script library used for prularizing words
   instaparse/instaparse                     {:mvn/version "1.4.12"}             ; Make your own parser
   clj-commons/clj-yaml                      {:mvn/version "1.0.27"}             ; Clojure wrapper for YAML library SnakeYAML
-  io.github.camsaul/toucan2                 {:mvn/version "1.0.537"}
-  io.github.eerohele/pp                     {:git/tag "2023-11-25.47"           ; super fast pretty-printing library
-                                             :git/sha "d3210d3"
+  io.github.camsaul/toucan2                 {:mvn/version "1.0.538"}
+  io.github.eerohele/pp                     {:git/tag "2024-01-04.60"           ; super fast pretty-printing library
+                                             :git/sha "a428751"
                                              :git/url "https://github.com/eerohele/pp"}
-  io.github.metabase/macaw                  {:mvn/version "0.1.21"}             ; Parse native SQL queries
+  io.github.metabase/macaw                  {:mvn/version "0.1.23"}             ; Parse native SQL queries
   ;; The 2.X line of Resilience4j requires Java 17, so we cannot upgrade this dependency until that is our minimum JVM version
-  io.github.resilience4j/resilience4j-retry {:mvn/version "1.7.1"}              ; Support for retrying operations
+  io.github.resilience4j/resilience4j-retry {:mvn/version "1.7.1" #_ "must be 1.7.1"} ; Support for retrying operations
```

Comment:
if you're using one of the auto-upgrade tools it can be easy to glaze over the comment on L81